hicexplorer/HiCMatrix.py

```diff
@@ -278,7 +282,7 @@ def getDekkerBins(self, fileName):
         self.header = []
         try:
             for line in gzip.open(fileName, 'r').readlines():
-                if line.startswith("#"):
+                if line.startswith(b"#"):
```

Comment:
I now found how we did it, use use some helper functions to take care of that. See this line: https://github.com/fidelram/deepTools/blob/master/deeptools/heatmapper.py#L748

Basically, each file line is transformed to a string at the beginning using this function: https://github.com/fidelram/deepTools/blob/0eb7a3a66eebd9ffe0e534d23ab7d5873fc5bfc2/deeptools/utilities.py#L285

I think both approaches are equally good. But I think that the specific decoding of the file contents as 'ascii' seems to take care in defining the type of data we want to use, instead of treating it as bytes.